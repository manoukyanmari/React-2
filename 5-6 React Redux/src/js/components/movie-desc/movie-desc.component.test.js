import React from 'react';
import { shallow } from 'enzyme';
import MovieDesc from './movie-desc';

describe('MovieDesc', () => {
    it('Adds MovieDesc snapshot testing', () => {
        const component = shallow(<MovieDesc/>);

        expect(component).toMatchSnapshot();
    });
});
