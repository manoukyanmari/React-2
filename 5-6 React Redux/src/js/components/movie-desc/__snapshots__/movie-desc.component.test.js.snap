// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDesc Adds MovieDesc snapshot testing 1`] = `ShallowWrapper {}`;
